Gemfile

```diff
@@ -16,7 +16,18 @@ gem 'acts_as_votable', '~> 0.10.0'
 gem 'simple_form'
 gem 'rails_12factor'
 gem "font-awesome-rails"
-gem 'cucumber'
+gem "pry"
+
+group :test, :development do
+  gem 'cucumber-rails', :require => false
+  # database_cleaner is not required, but highly recommended
+  gem 'database_cleaner'
+end
+
+group :development, :test do
```

Comment:
This group is duplicate of the above group declaration, just the symbols are in reverse order :)